.travis/build.sh

```diff
@@ -43,8 +43,51 @@ cat > pytest.ini << EOF
 addopts=-p no:logging --color=no --force-flaky
 EOF
 
+if [ "$TARGET_HOST" == "arm-linux-gnueabihf" ] || [ "$TARGET_HOST" == "arm-linux-gnueabihf" ]
+then
+    export QEMU_LD_PREFIX=/usr/"$TARGET_HOST"/
+    export MAKE_HOST="$TARGET_HOST"
+    export AR="$TARGET_HOST"-ar
+    export AS="$TARGET_HOST"-as
+    export CC="$TARGET_HOST"-gcc
+    export CXX="$TARGET_HOST"-g++
+    export LD="$TARGET_HOST"-ld
+    export STRIP="$TARGET_HOST"-strip
 
-if [ "$SOURCE_CHECK_ONLY" == "false" ]; then
+    wget -q https://zlib.net/zlib-1.2.11.tar.gz \
+    && tar xvf zlib-1.2.11.tar.gz \
+    && cd zlib-1.2.11 \
+    && ./configure --prefix="$QEMU_LD_PREFIX" \
+    && make \
+    && sudo make install 
+    cd .. && rm zlib-1.2.11.tar.gz && rm -rf zlib-1.2.11
```

Comment:
This sort of chaining is not necessary since we are using `set -e` in the script header:

```
      -e  Exit immediately if a command exits with a non-zero status.
```